src/commands/lightning/dev/component.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { SfCommand, Flags } from '@salesforce/sf-plugins-core';
+import { Messages } from '@salesforce/core';
+import { cmpDev } from '@lwrjs/api';
+
+Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
+const messages = Messages.loadMessages('@salesforce/plugin-lightning-dev', 'lightning.dev.component');
+
+export type LightningDevComponentResult = {
+  path: string;
+};
+
+export default class LightningDevComponent extends SfCommand<LightningDevComponentResult> {
+  public static readonly summary = messages.getMessage('summary');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessages('examples');
+
+  public static readonly flags = {
+    name: Flags.string({
+      summary: messages.getMessage('flags.name.summary'),
+      description: messages.getMessage('flags.name.description'),
+      char: 'n',
+      required: false,
+    }),
+    // TODO should this be required or optional?
```

Comment:
i vote optional

Reply:
Will let Brian decide based off what he has time to implement. If we don't make it required, will need to do some code scanning of the component u are attempting to preview and error out if it needs data, but you don't supply an authenticated org. Just added todo comment as a reminder